packages/gtkmm4.rb

```diff
@@ -3,23 +3,34 @@
 class Gtkmm4 < Meson
   description 'The Gtkmm4 package provides a C++ interface to GTK+ 4.'
   homepage 'https://www.gtkmm.org/'
-  version '4.0.2'
+  version '4.13.3'
   license 'LGPL-2.1+'
   compatibility 'x86_64 aarch64 armv7l'
-  source_url 'https://download.gnome.org/sources/gtkmm/4.0/gtkmm-4.0.2.tar.xz'
-  source_sha256 '0c836e8daffd836ef469499b7a733afda3a5260ea0e4d81c552f688ae384bd97'
+  source_url 'https://gitlab.gnome.org/GNOME/gtkmm.git'
+  git_hashtag version
   binary_compression 'tar.zst'
 
   binary_sha256({
-    aarch64: '9769d2f5e99b80ee126723b0614a2a5cad5298a6c14176644c9d7c7408fceb43',
-     armv7l: '9769d2f5e99b80ee126723b0614a2a5cad5298a6c14176644c9d7c7408fceb43',
-     x86_64: 'cb5053d74804b963f9fab9e9f97b336776cad30d6b38df277386e800b755c6b3'
+    aarch64: '181cd9d9c4dca2a443121ed4fd82e9f5ba249423e1777f8842700dbe1afefe8e',
+     armv7l: '181cd9d9c4dca2a443121ed4fd82e9f5ba249423e1777f8842700dbe1afefe8e',
+     x86_64: 'eb407224748eef793e5bed1271239d23520e50f7e1112c1b5a429a81c6ad0af0'
   })
 
-  depends_on 'atkmm'
-  depends_on 'gtk4'
-  depends_on 'pangomm'
-  depends_on 'cairomm'
-  depends_on 'vulkan_headers'
-  depends_on 'gobject_introspection'
+  depends_on 'atkmm' => :build
+  depends_on 'cairomm_1_16' # R
+  depends_on 'cairomm' => :build
+  depends_on 'cairo' # R
+  depends_on 'gcc_lib' # R
+  depends_on 'gdk_pixbuf' # R
+  depends_on 'glibc' # R
+  depends_on 'glibmm_2_68' # R
+  depends_on 'glib' # R
+  depends_on 'gobject_introspection' => :build
+  depends_on 'graphene' # R
+  depends_on 'gtk4' => :build
+  depends_on 'gtk4' # R
```

Comment:
`gtk4` is listed twice.

Reply:
Good catch! The new build system automatically adds the runtime dep if missing! Hmm. Maybe I will write code at some point to also switch the build dep to runtime dep if it notices the build dep is already there.